mantle/cmd/kola/kola.go

```diff
@@ -648,7 +649,17 @@ func getParentFcosBuildBase(stream string) (string, error) {
 			return getParentFcosBuildBase("testing-devel")
 		}
 
-		parentVersion = index.Releases[n-1].Version
+		findParentVersion:
+  		for i := range index.Releases {
+				//to iterate through the list in reverse order
+    		release := index.Releases[(n - 1 - i)]
+      	for _, commit := range release.Commits {
+    			if commit.Architecture == (system.RpmArch()) {
+    				parentVersion = release.Version
+      			break findParentVersion
+          }
+        }
+	    }
```

Comment:
If we get to this point and we have found no parent version then we need to output a message and return like we do above in the `n == 0` case. Maybe we can even output the same message and deduplicate the code between the two.

Reply:
addressed in recent commit. could you please have a look?